lib/cli/cli.js

```diff
@@ -547,6 +547,11 @@ module.exports.parseCommandLine = function parseCommandLine() {
         'Send the tags as Array or a String. In Graphite 1.0 the tags is a array. Before a String',
       group: 'Graphite'
     })
+    .option('graphite.annotationTitle', {
+      describe:
+        'Add a title to the annotation sent for a run. The message is attached after the default message and can contain HTML.',
```

Comment:
> The message is attached after the default message and can contain HTML

Copy/paste right? Its not attached to the default message? I'm not sure if it can contain HTML?

```suggestion
        'Add a title to the annotation sent for a run.',
```